lib/node_modules/@stdlib/math/base/special/gamma-lanczos-sum/src/main.c

```diff
@@ -0,0 +1,74 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/gamma_lanczos_sum.h"
+#include "stdlib/constants/float64/pinf.h"
+
+/**
+* Calculates the Lanczos sum approximation.
+*
+* @param {number} x - input value
+* @returns {number} Lanczos sum approximation
+*
+* @example
+* var v = gammaLanczosSum( 4.0 );
+* // returns ~950.366
+*
+* @example
+* var v = gammaLanczosSum( -1.5 );
+* // returns ~1373366.245
+*
+* @example
+* var v = gammaLanczosSum( -0.5 );
+* // returns ~-699841.735
+*
+* @example
+* var v = gammaLanczosSum( 0.5 );
+* // returns ~96074.186
+*
+* @example
+* var v = gammaLanczosSum( 0.0 );
+* // returns Infinity
+*
+* @example
+* var v = gammaLanczosSum( NaN );
+* // returns NaN
+*/
+double stdlib_base_gamma_lanczos_sum( const double x ) {
+	double ax;
+	double s1;
+	double s2;
+	double x1;
+	if( x == 0.0 ) {
+		return STDLIB_CONSTANT_FLOAT64_PINF;
+	}
+	if ( x < 0.0 ) {
+		ax = -x;
+	} else {
+		ax = x;
+	}
+	if ( ax <= 1.0 ) {
+		s1 = 38474670393.31777 + (x * (36857665043.51951 + (x * (15889202453.72942 + (x * (4059208354.298835 + (x * (680547661.1834733 + (x * (78239755.00312005 + (x * (6246580.776401795 + (x * (341986.3488721347 + (x * (12287.194511824551 + (x * (261.61404416416684 + (x * 2.5066282746310007))))))))))))))))))); // eslint-disable-line max-len
```

Comment:
1. We don't lint with ESLint.
2. You've inlined the evalrational implementation. We don't want to do this, as now the coefficients are hardcoded. Instead, update the script in the scripts directory to inject an `evalrational` function, as done in other packages. This function then becomes a one-liner calling that evalrational function. A decent C compiler will handle inlining.